packages/react-query-sdk/src/notifications/services/AlertsService.ts

```
@@ -0,0 +1,999 @@
+  UseQueryResult,
+  UseSuspenseInfiniteQueryOptions,
+  UseSuspenseInfiniteQueryResult,
+  UseSuspenseQueryOptions,
+  UseSuspenseQueryResult
+} from "@tanstack/react-query";
+
+import type { paths } from "../schema";
+export interface AlertsService {
+  createAlert: {
+    /**/
+    getMutationKey(parameters: DeepReadonly<CreateAlertParameters> | void): ServiceOperationMutationKey<CreateAlertSchema, CreateAlertParameters>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)